projectManager/projectManager-war/src/main/java/org/silverpeas/components/projectmanager/servlets/AjaxProjectManagerServlet.java

```diff
@@ -49,92 +49,67 @@
   private static final long serialVersionUID = 798968548064856822L;
   private static final String ACTION_LOAD_TASK = "loadTask";
   private static final String ACTION_COLLAPSE_TASK = "collapseTask";
+  private static final String REQUEST_PARAM_TASKID = "TaskId";
+  private static final String REQUEST_PARAM_BEGINDATE = "BeginDate";
 
   @Override
-  public void doGet(HttpServletRequest req, HttpServletResponse res)
-      throws ServletException, IOException {
+  public void doGet(HttpServletRequest req, HttpServletResponse res) throws IOException {
     doPost(req, res);
   }
 
   @Override
-  public void doPost(HttpServletRequest req, HttpServletResponse res)
-      throws ServletException, IOException {
+  public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 22 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS3776)